inputs/templates/terra_workspaces/cohort_mode/workflow_configurations/ClusterBatch.json.tmpl

```diff
@@ -27,5 +27,6 @@
   "ClusterBatch.wham_vcf_tar": "${this.std_wham_vcf_tar}",
   "ClusterBatch.manta_vcf_tar": "${this.std_manta_vcf_tar}",
   "ClusterBatch.melt_vcf_tar": "${this.std_melt_vcf_tar}",
+  "ClusterBatch.scramble_vcf_tar": "${this.std_scramble_vcf_tar}",
```

Comment:
We don't want to actually add scramble to any of the templates until we've evaluated it more thoroughly (also when we do migrate to scramble we'll want to disable melt and delete the melt inputs across workflows). Can you revert the changes to all `.tmpl` files? This includes the update to the terra workspace variable definitions.

Reply:
I changed the `.tmpl` following @epiercehoffman suggestions and I think the motivation was to enable running scramble benchmarks on Terra. Happy to revert these changes if we want to hold off on them for a while.

Reply:
@mwalker174 I want to add scramble to the Terra templates except for GatherSampleEvidence, so that it can be toggled on/off just by adding/removing the scramble docker in GatherSampleEvidence, just like MELT. But I agree we don't want to add it to the test JSONs just yet, so please revert the changes to `inputs/templates/test/EvidenceQC/EvidenceQC.json.tmpl`.

Reply:
@VJalili Just discussed this with emma. Fine to keep these Terra workflow configs in place since it doesn't actually enable scramble (ie providing the scramble docker in GatherSampleEvidence).